.github/workflows/build-doc.yml

```
@@ -21,12 +21,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.7
+    - name: Set up Python 3.8
       uses: s-weigand/setup-conda@v1
       with:
         update-conda: true
         conda-channels: conda-forge
-        python-version: 3.7
+        python-version: 3.8
```

By the time we enter `python-version: 3.10`, this YAML will parse to the number `3.1` and then we are in trouble :)

```suggestion
        python-version: "3.8"
```